File: Accenture/11.EvenOdd/code.cpp
#include <bits/stdc++.h>
using namespace std;
void label_even_odd(int A[], int N) {
    for (int i = 0; i < N; i++) {
        if (A[i] % 2 == 0) {
            cout << "even ";
        } else {
            cout << "odd ";
        }
    }
}
int main() {
    int N;
    cin >> N;
    int A[N];
    for (int i = 0; i < N; i++) {
        cin >> A[i];
    }
    label_even_odd(A, N);
}

File: Accenture/11.EvenOdd/code.py
def label_even_odd(A, N):
    for i in range(N):
        if A[i] % 2 == 0:
            print("even", end=" ")
        else:
            print("odd", end=" ")

N = int(input())
A = list(map(int, input().split()))
label_even_odd(A, N)
